encoding/encode.go

```diff
@@ -18,9 +18,19 @@ package encoding
 
 import (
 	"encoding/hex"
-	"errors"
 	"fmt"
 	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+type convertFunc func(int64) string
```

Comment:
I think we can have better name than `convertFunc`

Reply:
i agree!! it needs to be named more understandable!